### PR TITLE
fix(source-containers): all cachito metadata

### PR DESCRIPTION
**Pull request by MartinBasti**
Exporting all cachito metadata caused failure in source contianer builds as it only expected 2 archives.

STONEBLD-2655

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [x] Python type annotations added to new code
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request has a link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

File: tests/plugins/test_fetch_sources.py [hunk 1]
```diff
@@ -47,10 +47,10 @@
...
                        {'name': 'second', 'url': 'second_url',
                         'archives': ['remote-source-second.bad', 'remote-source-second.tar.gz']}]
-RS_TYPEINFO_NO_2 = [{'name': 'first', 'url': 'first_url',
-                     'archives': ['remote-source-first.tar.gz']},
-                    {'name': 'second', 'url': 'second_url',
-                     'archives': ['remote-source-second.tar.gz']}]
+RS_TYPEINFO_NO_TAR_GZ = [{'name': 'first', 'url': 'first_url',
+                         'archives': ['remote-source-first.json']},
+                         {'name': 'second', 'url': 'second_url',
+                         'archives': ['remote-source-second.json']}]
```
Comment (chmeliik) [inline tests/plugins/test_fetch_sources.py:53]: Would a test case with `.config.json` and `.env.json` be worth it?

**Review by chmeliik — APPROVED** [review]
LGTM, extra testcase optional

File: tests/plugins/test_fetch_sources.py (hunk 1 above)
Comment (MartinBasti) [reply tests/plugins/test_fetch_sources.py:53]: right, I can add it there to check if they are safely ignored

**Review by mkosiarc — APPROVED** [review]

File: tests/plugins/test_fetch_sources.py (hunk 1 above)
Comment (MartinBasti) [reply tests/plugins/test_fetch_sources.py]: added

**Review by chmeliik — APPROVED** [review]

**Pull request merged**